### DISCUSSION TITLE
AttributeError: module 'captcha_admin.admin' has no attribute 'ModelAdmin'

### DISCUSSION THREAD

**Discussion by DiAlCuSa** [django-recaptcha/django-recaptcha#266] [Q&A] [answered]
I would like to know how to solve this error

**Comment by Bishwas-py** [comment, marked as answer]
I think you wanna check this article to; [Django Recaptcha version 3](https://webmatrices.com/post/django-recaptcha-v3-install-recaptcha-in-django-site-28)
